CMakeLists.txt

```diff
@@ -100,14 +100,28 @@ unset(MEMCHECK CACHE)
 
 # BINDINGS_PYTHON ==> LIBCELLML_BINDINGS_PYTHON
 set(_PARAM_ANNOTATION "Build Python wrappers.")
-if(BINDINGS_AVAILABLE)
+if(BINDINGS_AVAILABLE AND Python_Development_FOUND)
   set(LIBCELLML_BINDINGS_PYTHON ON CACHE BOOL ${_PARAM_ANNOTATION})
 endif()
-if(DEFINED BINDINGS_PYTHON AND BINDINGS_AVAILABLE)
+if(DEFINED BINDINGS_PYTHON AND BINDINGS_AVAILABLE AND Python_Development_FOUND)
   set(LIBCELLML_BINDINGS_PYTHON "${BINDINGS_PYTHON}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
+elseif(BINDINGS_PYTHON)
+  message(WARNING "Python bindings requested but development files for Python or swig was not found!")
 endif()
 unset(BINDINGS_PYTHON CACHE)
 
+# PYTHON_COVERAGE ==> LIBCELLML_PYTHON_COVERAGE
+set(_PARAM_ANNOTATION "Enable Python coverage testing.")
+if(PYTHON_COVERAGE_TESTING_AVAILABLE)
+  set(LIBCELLML_PYTHON_COVERAGE ON CACHE BOOL ${_PARAM_ANNOTATION})
+endif()
+if(DEFINED PYTHON_COVERAGE AND PYTHON_COVERAGE_TESTING_AVAILABLE)
+  set(LIBCELLML_PYTHON_COVERAGE "${PYTHON_COVERAGE}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
+elseif(PYTHON_COVERAGE)
+  message(WARNING "Python coverage testing requested but the requirements were not found!")
```

Comment:
Such a warning wouldn't help me unless I was to check this file. Might be worth mentioning what those requirements are? (Same elsewhere maybe?)